apps/contracts/contracts/Zengo.sol

```diff
@@ -1,16 +1,17 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.17;
 
-// import "@thirdweb-dev/contracts/extension/PermissionsEnumerable.sol";
-// import "@thirdweb-dev/contracts/extension/ContractMetadata.sol";
+import "@thirdweb-dev/contracts/extension/PermissionsEnumerable.sol";
+import "@thirdweb-dev/contracts/extension/ContractMetadata.sol";
 import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "./interfaces/IStates.sol";
 import "./Constants.sol";
 import "./lib/Structs.sol";
 import "./storage/ZengoStorage.sol";
 
-contract ZengoDAO is Constants, ZengoStorage {
+contract ZengoDAO is Constants, ZengoStorage, PermissionsEnumerable, ContractMetadata {
     IERC20 public token;
```

Comment:
is the token going to be used? 

For example,  testing, to deploy it the constructor asks for the token

Reply:
The token is the ERC20 that is going to be distributed during the funding round

Reply:
> The token is the ERC20 that is going to be distributed during the funding round

Ah, and that address is needed to set it at the deployment ? Is this going to be a custom ERC20 ? or DAI/USDT for example?

Reply:
We can use any address we want to

Reply:
> > > The token is the ERC20 that is going to be distributed during the funding round
> > 
> > Ah, and that address is needed to set it at the deployment ? Is this going to be a custom ERC20 ? or DAI/USDT for example?
> > 
> We can use any address we want to

and that address is needed to set at the deployment ?

Reply:
Yes